python/pyarrow/tests/test_parquet.py

```diff
@@ -3015,7 +3015,6 @@ def test_dictionary_array_automatically_read():
     assert result.schema.metadata is None
 
 
-@pytest.mark.pandas
```

Comment:
in the rest of the file, test functions using pandas are marked as such?

Reply:
Yes, this is a rebase artifact, fixing